docs/conf.py

```diff
@@ -62,25 +64,25 @@ def __getattr__(cls, name):
 master_doc = "index"
 
 # General information about the project.
-project = u"FreeGS"
-copyright = u"2017-2019, Ben Dudson"
-author = u"Ben Dudson"
+project = "FreeGS"
+copyright = "2017-2023, Ben Dudson"
+author = "Ben Dudson"
```

Comment:
Should we list everyone, or refer to the repository to see the authors?

Reply:
I count 14 contributors, probably a bit too many for here?

Reply:
Just "FreeGS authors"?

Reply:
Done!